tests/FunctionsTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Tests;
+
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use functions\functions;
+
+
+class FunctionsTest extends TestCase
+{
+    protected Functions $function;
+
+    protected function setUp(): void
+    {
+        $this->function = new Functions();
+    }
+
+    public function testSayHelloPositive()
+    {
+        $this->assertEquals('Hello', $this->function->sayHello());
+    }
+
+    /**
+     * @dataProvider positiveSayHelloArgumentDataProvider
+     */
+    public function testSayHelloArgumentPositive($input, $expected)
+    {
+        $this->assertEquals($expected, $this->function->sayHelloArgument($input));
+    }
+
+    public function positiveSayHelloArgumentDataProvider(): array
+    {
+        return [
+            ['world!', 'Hello world!'],
+            ['', 'Hello '],
+            ['світ', 'Hello світ'],
+            [1, 'Hello 1'],
+            [1.2, 'Hello 1.2'],
+        ];
+    }
+
+    public function testSayHelloArgumentWrapperNegative()
```

Comment:
Please rename it to something like `testSayHelloArgumentWrapperThrowExceptionWhenPassingArray`